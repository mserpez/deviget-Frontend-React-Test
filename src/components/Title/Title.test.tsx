import React from 'react';
import { shallow } from 'enzyme';
import { Title } from './';

test('Renders the component', () => {
    const component = shallow(<Title label="Test" />)
    expect(component).toMatchSnapshot();
    expect(component.text()).toBe("Test");
});