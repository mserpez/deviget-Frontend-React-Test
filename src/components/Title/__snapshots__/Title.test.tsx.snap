// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders the component 1`] = `
<h1>
  Test
</h1>
`;
